cloudevents/working-drafts/xml-format.md

```diff
@@ -0,0 +1,360 @@
+# XML Event Format for CloudEvents
+
+## Abstract
+
+This format specification for CloudEvents defines how events are expressed
+in XML documents.
+
+## Status of this document
+
+This document is a working draft.
+
+## Table of Contents
+
+1. [Introduction](#1-introduction)
+2. [Attributes](#2-attributes)
+3. [Data](#3-data)
+4. [Envelope](#4-envelope)
+5. [XML Batch Format](#5-xml-batch-format)
+6. [Examples](#6-examples)
+
+## 1. Introduction
+
+[CloudEvents][ce-spec] is a standardized and protocol-agnostic definition of the
+structure and metadata description of events. This specification defines how
+elements defined in the CloudEvents specification are to be represented using
+[Extensible Markup Language (XML)](xml-spec) documents.
+
+* The [Attributes](#2-attributes) section describes the representation and
+data type mappings for CloudEvents context attributes.
+
+* The [Data](#3-data) section defines the container for the data portion of a
+CloudEvent.
+
+* The [Envelope](#4-envelope) section defines an XML element for CloudEvents
+attributes and an associated media type.
+
+* The [Batch](#5-xml-batch-format) section describes how multiple CloudEvents
+can be packaged into a single XML document.
```

Comment:
@JemDay your comment in the previous (resolved) comment seemed to indicate you wanted to switch to 'element'. Did you change your mind?

Reply:
I probably missed that one .. I'll flip it to `element`.

This has the potential to be a little "trippy" .. you can create a `<batch>..</batch>` and put it in the `data` of another CloudEvent .. in fact you could nest them many times, just like a nesting doll ;-)